include/tvm/ir/transform.h

```diff
@@ -410,9 +410,30 @@ class Sequential : public Pass {
  *
  * \return The created module pass.
  */
-TVM_DLL Pass
-CreateModulePass(const runtime::TypedPackedFunc<IRModule(IRModule, PassContext)>& pass_func,
-                 int opt_level, String name, Array<runtime::String> required);
+Pass CreateModulePass_(const runtime::TypedPackedFunc<IRModule(IRModule, PassContext)>& pass_func,
+                       int opt_level, String name, Array<runtime::String> required);
+
+/*
+ * \brief Create a module pass.
+ *
+ * \param pass_func The function that contains the optimization.
+ * \param opt_level The optimization level of the module pass.
```

Comment:
It would be great to leave these in a separate PR. Given the particular change will affect all PackedFunc calls. A smaller set of changes would be easier for others to review and take a look at

Reply:
I've reverted these changes and will push them in a subsequent PR.